docs/changelog.md

```diff
@@ -11,18 +11,32 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - Experimental support for Vue
 - Updated the `six-item-picker` component to include tabindex attributes for better accessibility.
 - Added focus-visible to `six-checkbox` component for better accessibility.
+- Added `matchTriggerWidht` property to `six-dropdown`.
+- Added `selectAllButton` property to `six-select`.
 
 ### Changed
 
 - Angular example upgraded to Angular 16
 - **Breaking**: Upgraded to Angular 16. Projects need to update to Angular 16 or newer.
+- **Breaking**: Part `tag` no longer supported in `six-select`. Selected items are shown as comma
+  separated values.
+- Moved check icons to the right in `six-select`.
+- Replaced check icons with checkboxes for `six-select` multiselect.
+- Display selected items on top of dropdown for `six-select` multiselect.
+- Reduced vertical spacing in `six-menu` items.
+- Improved visuals of `six-dropdonw` and added search icon to filter input.
+- Changed default debounce value to 0 for `six-dropdown` filter.
+- Either show clear or expand icon in `six-select`, but never both.
+- **Breaking**: Merged shadow css variables `--six-elevation-` into `--six-shadow-` and reduced
+  variants to small, medium and large.
```

Comment:
I appreciate the hard work that has gone into this extensive PR, however to enhance the review efficiency and reduce the risk of breaking things, I'd suggest splitting such big PRs into smaller, more focused ones ☺

Reply:
I absolutely understand your concerns regarding the size of the PR. Unfortunately, it is extremely difficult to split this PR into smaller ones because the changes are distributed across multiple components, including select, dropdown, menu, input and menu-item. Even though it could be done now, after the changes have already been made, it would take a considerable amount of time to reorganize the PR.

That said, I have tried to group the changes into single commits, which should hopefully help a little. The last few commits are not grouped as they fix issues identified in earlier commits, and it is now a bit difficult to rebase them.